### PR TITLE
chore: update packages and update code into apify-client v2

### LINKED ISSUE #22: Migrate to apify-client-js v3
All code and tests need to be rewritten into apify-client-js v1 client.

### PR DESCRIPTION
**Pull request by drobnikj**
fixes #22

### CODE REVIEW

**Review by fnesveda — APPROVED** [review]
The changes look good, I was just thinking that most of the methods in `apify_helpers` can be replaced with methods from `ApifyClient`, since the new version has them. Also we should get rid of `bluebird`, that's a blast from the past 😄 

Should we do it in another PR?

File: test/triggers/task_run_finished.js [hunk 1]
```diff
@@ -113,17 +108,16 @@ describe('task run finished trigger', () => {
 
         expect(results.length).to.be.eql(3);
         expect(results[0].id).to.be.eql(runs.pop().id);
-        expect(results[0]).to.have.all.keys(Object.keys(TASK_RUN_SAMPLE));
+        expect(results[0]).to.have.all.keys(Object.keys(TASK_RUN_SAMPLE).concat(['isStatusMessageTerminal', 'statusMessage']));
```
Comment (valekjo) [inline test/triggers/task_run_finished.js:111]: Maybe the `TASK_RUN_SAMPLE` should be extended?

**Review by valekjo — APPROVED** [review]
One comment, looks good otherwise.

File: test/triggers/task_run_finished.js (hunk 1 above)
Comment (drobnikj) [reply test/triggers/task_run_finished.js:111]: It is not possible as these attributes are there just a few runs not fo all of them.

**Comment by drobnikj** [comment]
> the methods in `apify_helpers` can be replaced

@fnesveda The problem here is that we need to use z.request which is passed there from methods in Zapier to be able do some metrics and debugging there.

**Pull request merged**